ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/security/auth-config.module.ts

```diff
@@ -0,0 +1,47 @@
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { APP_INITIALIZER, NgModule } from '@angular/core';
+import { AuthInterceptor, AuthModule, LogLevel, OidcConfigService, OidcSecurityService } from 'angular-auth-oidc-client';
+import { ConfigService } from 'src/app/services/config.service';
+import { RefreshTokenParameterInterceptor } from './refresh-token-parameter.interceptor';
+
+export function loadConfig(configService: ConfigService, oidcConfigService: OidcConfigService): Function {
+    return () => {
+        configService.getClientSettings().subscribe(clientSettings => {
+            // https://github.com/damienbod/angular-auth-oidc-client/blob/8b66484755ad815948d5bc0711e8d9c69ac6661f/docs/configuration.md
+            oidcConfigService.withConfig({
+                stsServer: `https://login.microsoftonline.com/${clientSettings.tenant_id}/v2.0`,
+                redirectUrl: clientSettings.redirect_uri,
+                postLogoutRedirectUri: clientSettings.post_logout_redirect_uri,
+                clientId: clientSettings.client_id,
+                scope: `openid profile api://${clientSettings.client_id}/feapi`,
+                responseType: 'code',
+                maxIdTokenIatOffsetAllowedInSeconds: 600,
+                autoUserinfo: false,
+                logLevel: LogLevel.Debug,
```

Comment:
don't need to have debug level in production mode. Please use the environment switch to turn off debug when in prod mode

Reply:
resolved myself